src/Resources/skeleton/Admin.tpl.php

```diff
@@ -13,13 +13,13 @@
 final class <?= $class_name ?> extends AbstractAdmin
```

Comment:
Why the admin are final by default ?
In our project we often extends our admin, and I feel like that making the class final by default is like saying "Extending an admin is a bad practice".